test/ecto_enum_migration_test.exs

```diff
@@ -130,6 +139,17 @@ defmodule EctoEnumMigrationTest do
         :ok = down(num, DropTypeWithCustomSchemaMigration)
       end
     end
+
+    test "supports drop if exists option" do
```

Comment:
Could we have a another test, for the case when the type exists and it gets deleted?

Reply:
Done @thiamsantos, thanks!